src/resources/espnow.cc

```diff
@@ -157,10 +191,10 @@ static void free_datagram(struct DataGram* datagram) {
 
 static void espnow_send_cb(const uint8_t *mac_addr, esp_now_send_status_t status) {
```

Comment:
```suggestion
static void espnow_send_cb(const uint8* mac_addr, esp_now_send_status_t status) {
```

Reply:
Leaving as is, at this code will be changed in a depending PR that fixes this.